### PR TITLE
Add a new language (russian) & add missing ukrainian strings

### PR DESCRIPTION
**Pull request by Melishy**
Glad to contribute it! Just i do speak russian, and i am doing this for russian peoples to understand.
*it's okay if you deny this*

### CODE REVIEW

File: common/src/main/resources/assets/bridgingmod/lang/ru_ru.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  "config.bridgingmod.category.vfx.tooltip": "Изменение внешнего вида всех индикаторов",
+  "config.bridgingmod.category.fixes": "Исправления",
+  "config.bridgingmod.category.fixes.tooltip": "Придирчивые вещи, которые не нужно менять, но они могли бы сделать мод лучше.",
+  "config.bridgingmod.category.debug": "Отлаживать",
+  "config.bridgingmod.category.debug.tooltip": "Функции, которые полезны для разработки, но могут оказаться полезными и вам.",
+  "config.bridgingmod.category.other": "Другой",
+  "config.bridgingmod.category.other.tooltip": "Разные настройки",
+  "config.bridgingmod.option.enableBridgingAssist": "Включить мост",
+  "config.bridgingmod.option.enableBridgingAssist.description.0": "Должны ли быть активными мосты, подобные коренной скале? \n",
+  "config.bridgingmod.option.enableBridgingAssist.description.1": "Это также привязано к привязке клавиш в элементах управления (по умолчанию ",")",
```
Comment (squeeglii) [inline common/src/main/resources/assets/bridgingmod/lang/ru_ru.json]: Line is broken at the end by `","`  - use single quote marks `'` inside the translation instead please, or use an escape (`\"`)

File: common/src/main/resources/assets/bridgingmod/lang/uk_ua.json [hunk 2]
```diff
@@ -1,41 +1,66 @@
...
-
-    "config.bridgingmod.option.showDebugHighlight": "Показати контур блоку містового асистента у F3",
-    "config.bridgingmod.option.showNonBridgingDebugHighlight": "Показати контур блоку без містового асистента у F3",
-    "config.bridgingmod.option.showDebugTrace": "Показувати слід блоку містового асистента у F3"
-}
\ No newline at end of file
+  "bridgingmod.config.failed_integration.title": "Інтерфейс конфігурації вимкнено!",
+  "bridgingmod.config.failed_integration.description": "Не вдалося відкрити інтерфейс конфігурації. ",
+  "key.categories.bridgingmod": "Bridging Mod",
+  "key.bridgingmod.toggle_bridging": "Перемкнути перемикання",
```
Comment (squeeglii) [inline common/src/main/resources/assets/bridgingmod/lang/uk_ua.json]: The old translation seemed more accurate when translated back to English with Google Translate.
This mentions bridging, right?

**Review by squeeglii — CHANGES_REQUESTED** [review]
Russian Translations are great! Thank you for those!
Ukranian Translations - were they particularly inaccurate before? Google Translate gave them the A-Okay last time so I'm a little confused as the old ones seem to match better. If you're sure they're 100% accurate to the English, let me know.

File: common/src/main/resources/assets/bridgingmod/lang/ru_ru.json (hunk 1 above)
Comment (Melishy) [reply common/src/main/resources/assets/bridgingmod/lang/ru_ru.json]: oops, my bad! let me fix that.

File: common/src/main/resources/assets/bridgingmod/lang/uk_ua.json (hunk 2 above)
Comment (Melishy) [reply common/src/main/resources/assets/bridgingmod/lang/uk_ua.json]: i wasn't having a idea if the mod name was supposed to be translated

Comment (Melishy) [reply common/src/main/resources/assets/bridgingmod/lang/uk_ua.json]: but some of strings translated to ukrainian are missing from original one, so i decided to

**Comment by Melishy** [comment]
alright, fixed strings.

**Comment by Melishy** [comment]
> Russian Translations are great! Thank you for those! Ukranian Translations - were they particularly inaccurate before? Google Translate gave them the A-Okay last time so I'm a little confused as the old ones seem to match better. If you're sure they're 100% accurate to the English, let me know.

It's accurate (and ukrainian translations), because i noticed some strings are missing for translate, like in config, at "Fixes" tab.

File: common/src/main/resources/assets/bridgingmod/lang/uk_ua.json (hunk 2 above)
Comment (squeeglii) [reply common/src/main/resources/assets/bridgingmod/lang/uk_ua.json]: oops, my bad. I didn't realise Github selected that.

I'm looking at `key.bridgingmod.toggle_bridging` which already existed. It's changed from "Перемикання мостіння" to "Перемкнути перемикання".

Comment (Melishy) [reply common/src/main/resources/assets/bridgingmod/lang/uk_ua.json]: hm, alright, i changed it back

**Comment by squeeglii** [comment]
Thank youuuuuu! :D

**Pull request merged**